src/commands/edge_app_utils.rs

```diff
@@ -168,29 +170,30 @@ pub fn generate_file_tree(files: &[EdgeAppFile], root_path: &Path) -> HashMap<St
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::commands::SettingType;
     use std::fs::File;
     use std::io::Write;
     use tempfile::tempdir;
 
     fn create_manifest() -> EdgeAppManifest {
         EdgeAppManifest {
             app_id: Some("01H2QZ6Z8WXWNDC0KQ198XCZEW".to_string()),
-            entrypoint: None,
-            user_version: "1".to_string(),
-            description: "asdf".to_string(),
-            icon: "asdf".to_string(),
-            author: "asdf".to_string(),
-            homepage_url: "asdfasdf".to_string(),
+            user_version: Some("1".to_string()),
+            description: Some("asdf".to_string()),
+            icon: Some("asdf".to_string()),
+            author: Some("asdf".to_string()),
+            homepage_url: Some("asdfasdf".to_string()),
+            entrypoint: Some("entrypoint.html".to_owned()),
```

Comment:
FYI: Entrypoint just like app_id can't be empty

Reply:
@sergey-borovkov shouldn't `entrypoint` be optional (and default to `index.html`)?

Reply:
@vpetersson right, but you can't just send an empty entrypoint. We will default to index.html if it's not sent. But we should not allow user to specify an empty string entrypoint - this just makes no sense. 
When you call create it will contain "index.html". If you remove it we will handle it in the player.

Reply:
Let's make a decision, making it required is as simple as `false` -> `true`

Reply:
I made `entrypoint` optional, but it fails validation when in manifest and empty